docs/dictionary/function/number.lcdoc

```diff
@@ -57,6 +59,9 @@ Any stack reference.
 textString (string):
 Any string or expression that evaluates to a string.
 
+array (array):
+The name of an array
```

Comment:
This should probably be the standard:
```
An array or expression that evaluates to an array
```
as it could be eg `put the number of elements in functionThatReturnsAnArray()`